pkg/ipcache/ipcache.go

```diff
@@ -589,6 +598,13 @@ func (ipc *IPCache) RemoveIdentityOverride(cidr netip.Prefix, identityLabels lab
 	ipc.RemoveMetadata(cidr, resource, overrideIdentity(true), identityLabels)
 }
 
+// WaitForRevision will block until the desired revision has been reached. It can be used
```

Comment:
Should we move/copy the `(*metadata).waitForRevision` godoc here? I think it's worth mentioning in the public API that it's waiting for the revision to be ≥ the desired revision. Also might be worth to rename the method to `WaitForRevisionGreaterEqual` (ref. my review comment on `(*metadata).waitForRevision`.

Reply:
Tightened up the wording a bit.